include/aspect/boundary_composition/interface.h

```diff
@@ -27,9 +27,13 @@
 #include <aspect/utilities.h>
 #include <aspect/simulator_access.h>
 
+#include <deal.II/base/std_cxx11/shared_ptr.h>
```

Comment:
we can now use ``std::shared_ptr``, see #2378. Can you please remove this header and replace the type below?